source/bsondump.txt

```diff
@@ -37,133 +41,25 @@ to format its data.
 
 .. include:: /includes/extracts/require-cmd-line-bsondump.rst
 
-Versioning
-~~~~~~~~~~
-
-.. include:: /includes/extracts/dbtools-version-single.rst
-
-.. note:: Quick links to older documentation
-
-   - :v4.2:`MongoDB 4.2 bsondump <reference/program/bsondump>`
-   - :v4.0:`MongoDB 4.0 bsondump <reference/program/bsondump>`
-
-This documentation is for version ``{+release+}`` of |tool-binary|. 
-
-Compatibility
--------------
+Compatibility and Installation
```

Comment:
(linking) 
Either add the Learn More section or include the section headings with links. Unnecessary to have both.

Reply:
Could we please use the "Learn More" pattern at the bottom of the other pages on the landing page?  Internal search boosts on headers, so we don't want to reuse them on this page when we have pages with those titles.